app/models/wrangling_guideline.rb

```diff
@@ -0,0 +1,14 @@
+class WranglingGuideline < ActiveRecord::Base
+  acts_as_list
+
+  attr_protected :content_sanitizer_version
+
+  validates_presence_of :content, :title
+  validates_length_of :content, :maximum => ArchiveConfig.CONTENT_MAX,
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Prefer the new style validations `validates :column, length: value` over `validates_length_of`.